package/MDAnalysis/analysis/contacts.py

```diff
@@ -584,19 +584,33 @@ def output_exists(self, force=False):
         """
         return os.path.isfile(self.output) and not (self.force or force)
 
-    def run(self, store=True, force=False, start_frame=1, end_frame=None, step_value=1):
+    def run(self, store=True, force=False, start=0, stop=None, step=1, **kwargs):
         """Analyze trajectory and produce timeseries.
 
         Stores results in :attr:`ContactAnalysis1.timeseries` (if store=True)
         and writes them to a data file. The average q is written to a second
         data file.
-        *start_frame*
-            The value of the first frame number in the trajectory to be used (default: frame 1)
-        *end_frame*
-            The value of the last frame number in the trajectory to be used (default: None -- use all frames)
-        *step_value*
+        *start*
+            The value of the first frame index in the trajectory to be used (default: index 0)
+        *stop*
+            The value of the last frame index in the trajectory to be used (default: None -- use all frames)
+        *step*
             The number of frames to skip during trajectory iteration (default: use every frame)
         """
+        import warnings
+
+        if 'start_frame' in kwargs:
+            warnings.warn("start_frame argument has been deprecated, use start instead")
```

Comment:
I nevertheless think that these should be `DeprecationWarning`, just because it's the right semantics.